client/components/inline-support-link/index.jsx

```diff
@@ -124,16 +117,11 @@ class InlineSupportLink extends Component {
 					this.props.routeModalData.openModal( supportPostId );
 					return openDialogReturn;
 				} }
-				onMouseEnter={
-					! isDefaultLocale( localeSlug ) && ! shouldLazyLoadAlternates
-						? this.loadAlternates
-						: undefined
-				}
+				onMouseEnter={ ! isDefaultLocale( localeSlug ) ? this.prefetchAlternates : undefined }
```

Comment:
This doesn't seem to be needed anymore since the prefetching has been removed.

Reply:
Good catch! Cleaned up further in https://github.com/Automattic/wp-calypso/pull/62161/commits/b36de018e0459e388199fc292f5bde84ca9da429